tweb/src/main/java/io/fundreqest/platform/tweb/request/RequestController.java

```diff
@@ -92,6 +95,17 @@ public ModelAndView details(@PathVariable Long id, Model model) {
                 .build();
     }
 
+    @GetMapping(value = "/requests/{id}/badge", produces = "image/svg+xml")
```

Comment:
Maybe check 
https://stackoverflow.com/questions/31025467/thymeleaf-second-resolver-for-svg-in-spring-boot